### DISCUSSION TITLE
MSI support for boxedwine

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#228] [Feature Requests]
It would be nice if there was support for .msi files to be used in boxedwine.

**Comment by DustinBrett** [comment]
I'll keep this in mind as a feature request but at the moment msi files do not work in BoxedWine afaik. A new release is coming next year that may have more support for more things. https://github.com/danoon2/Boxedwine/issues/45#issuecomment-1279992581
